core/src/foundations/ionic.vars.scss

```
@@ -939,6 +367,578 @@ $ion-primitives-pink-1100: var(--ion-primitives-pink-1100, #8b2248);
+$ion-border-warning: var(--ion-border-warning, $ion-semantics-warning-900);
+$ion-border-warning-rgb: var(--ion-border-warning-rgb, 150, 101, 3);
+$ion-border-disabled: var(--ion-border-disabled, $ion-primitives-neutral-300);
+$ion-border-disabled-rgb: var(--ion-border-disabled-rgb, 224, 224, 224);
+$ion-border-focus-0: var(--ion-border-focus-0, none);
+$ion-border-focus-default: var(--ion-border-focus-default, $ion-primitives-blue-400);
+$ion-border-focus-default-rgb: var(--ion-border-focus-default-rgb, 181, 192, 247);
+$ion-border-focus-error: var(--ion-border-focus-error, $ion-primitives-red-400);
+$ion-border-focus-error-rgb: var(--ion-border-focus-error-rgb, 250, 161, 161);
+$ion-border-danger-default: var(--ion-border-danger-default, $ion-semantics-danger-800);
+$ion-border-danger-default-rgb: var(--ion-border-danger-default-rgb, 191, 34, 34);
+$ion-border-danger-press: var(--ion-border-danger-press, $ion-semantics-danger-1000);
+$ion-border-danger-press-rgb: var(--ion-border-danger-press-rgb, 118, 21, 21);
+$ion-border-input-default: var(--ion-border-input-default, $ion-primitives-neutral-500);
+$ion-border-input-default-rgb: var(--ion-border-input-default-rgb, 162, 162, 162);
+$ion-border-input-press: var(--ion-border-input-press, $ion-primitives-neutral-700);
+$ion-border-input-press-rgb: var(--ion-border-input-press-rgb, 119, 119, 119);
+$ion-border-input-read-only: var(--ion-border-input-read-only, $ion-primitives-neutral-500);
+$ion-border-input-read-only-rgb: var(--ion-border-input-read-only-rgb, 162, 162, 162);
+$ion-border-style-none: var(--ion-border-style-none, none);
+$ion-border-style-solid: var(--ion-border-style-solid, solid);
+$ion-border-style-dashed: var(--ion-border-style-dashed, dashed);
+$ion-border-style-dotted: var(--ion-border-style-dotted, dotted);
+$ion-border-size-0: var(--ion-border-size-0, $ion-scale-0);
+$ion-border-size-025: var(--ion-border-size-025, $ion-scale-025);
+$ion-border-size-050: var(--ion-border-size-050, $ion-scale-050);
+$ion-border-size-075: var(--ion-border-size-075, $ion-scale-075);
+$ion-border-radius-0: var(--ion-border-radius-0, $ion-scale-0);
+$ion-border-radius-100: var(--ion-border-radius-100, $ion-scale-100);
+$ion-border-radius-200: var(--ion-border-radius-200, $ion-scale-200);
+$ion-border-radius-300: var(--ion-border-radius-300, $ion-scale-300);
+$ion-border-radius-400: var(--ion-border-radius-400, $ion-scale-400);
+$ion-border-radius-500: var(--ion-border-radius-500, $ion-scale-500);
+$ion-border-radius-800: var(--ion-border-radius-800, $ion-scale-800);
+$ion-border-radius-1000: var(--ion-border-radius-1000, $ion-scale-1000);
+$ion-border-radius-full: var(--ion-border-radius-full, 999px);
+$ion-border-radius-025: var(--ion-border-radius-025, $ion-scale-050);
+$ion-bg-primary-base-default: var(--ion-bg-primary-base-default, $ion-semantics-primary-base);
+$ion-bg-primary-base-default-rgb: var(--ion-bg-primary-base-default-rgb, 16, 92, 239);
```

```suggestion
$ion-bg-primary-base-default-rgb: var(--ion-bg-primary-base-default-rgb, $ion-semantics-primary-base-rgb);
```

Should this fallback to `$ion-semantics-primary-base-rgb` (and the same for all other bg rgb vars)?